packages/next/src/build/analysis/get-page-static-info.ts

```diff
@@ -74,7 +75,15 @@ export function getRSCModuleInformation(
   const clientEntryType = clientInfoMatch?.[2] as 'cjs' | 'auto'
 
   const type = clientRefs ? RSC_MODULE_TYPES.client : RSC_MODULE_TYPES.server
-  return { type, actions, clientRefs, clientEntryType, isClientRef }
+  const hasUseClientDirective = /^\s*['"]use client['"]/.test(source)
```

Comment:
@huozhi @shuding I add this check since files with `'use client'` had the `type: 'server'`. 

Is that expected?